po/uk_UA.po

```diff
@@ -0,0 +1,221 @@
+# Ukrainian translation for Recorder.
+# Copyright (C) 2024 THE Recorder'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the Recorder package.
+# volkov <volkovissocool@gmail.com>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Recorder\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-01-06 12:52+0800\n"
+"PO-Revision-Date: 2024-01-07 15:52+0200\n"
+"Last-Translator: volkov <volkovissocool@gmail.com>\n"
+"Language-Team: volkov <volkovissocool@gmail.com>\n"
+"Language: uk_UA\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
+"n%10<=4 && (n%100<12 || n%100>14) ? 1 : 2);\n"
+"X-Generator: Poedit 3.4\n"
+"X-Poedit-Basepath: ..\n"
+"X-Poedit-SearchPath-0: src/MainWindow.vala\n"
+"X-Poedit-SearchPath-1: data/com.github.alexkdeveloper.recorder.appdata.xml."
+"in\n"
+"X-Poedit-SearchPath-2: data/com.github.alexkdeveloper.recorder.desktop.in\n"
+
+#: data/com.github.alexkdeveloper.recorder.appdata.xml.in:7
+#: data/com.github.alexkdeveloper.recorder.desktop.in:4
+msgid "Recorder"
+msgstr "Диктофон"
```

Comment:
Maybe it would be better to leave this line untranslated?
The program does not record audio from the microphone, but only the audio stream of the desktop.

Reply:
Welp, there also variant using "Записувач" which is more direct translation for "Recorder"

Reply:
Yes, I think this option is more appropriate. Thank you for your contribution)